### PR TITLE
Redirect to the address picker page if the user has not selected a location

### PR DESCRIPTION
**Pull request by mikemorreale**
...cation

### CODE REVIEW

File: src/requireLocation.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const requireLocation = (store: any) => {
+  return (nextState: any, replace: any) => {
+    if (!store.getState().user.location) {
```
Comment (matt-casey) [inline src/requireLocation.ts:3]: we should pull in a selector from src/modules/user here to get location

File: src/requireLocation.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+      replace({
+        pathname: '/',
+        state: { nextPathname: nextState.location.pathname }
```
Comment (matt-casey) [inline src/requireLocation.ts]: is this so that we can continue the user on to their attempted page after completing popup?

Comment (mikemorreale) [reply src/requireLocation.ts]: Currently this just redirects to the home page.

Comment (matt-casey) [reply src/requireLocation.ts]: right, but the "state: { nextPathname: nextState.location.pathname }" part, what is nextPathname used for?

**Comment by matt-casey** [comment]
LGTM

File: src/requireLocation.ts (hunk 2 above)
Comment (mikemorreale) [reply src/requireLocation.ts]: Oh, yeah we don't need that. Also, by removing that the query param no longer appears in the URL.

**Comment by mikemorreale** [comment]
I removed that unnecessary line for now.

**Pull request merged**